### PR TITLE
finish doc with npm

### PR DESCRIPTION
**Pull request by vagarii**

### CODE REVIEW

File: src/components/Box/LoadingBubble.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  max-width: 78%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-bottom-left-radius: 0;
+  border: ${bubbleColor === '#fff' ? 'solid 1px #D9D9D9' : null};
+`;
+
+  const wave = keyframes`
+    // 0%, 50%, 100%  {
```
Comment (olgafedorenko) [inline src/components/Box/LoadingBubble.js:48]: please delete unnecessary comments

File: src/components/Box/TextBubble.js [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+  display: inline-block;
+  width: auto;
+	max-width: 78%;
+	border-bottom-left-radius: ${props => props.withTail ? '0' : null};;
+	word-wrap: break-word;
+`;
+
+const Image = styled.img`
+  height: ${(props) => {
+		if (!props.withTail) {
```
Comment (olgafedorenko) [inline src/components/Box/TextBubble.js:60]: please remove extra space

File: src/components/Box/TextBubble.js [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+
+  if (step.pdf) {
+    displayImg = (
+      <Document file={image} style={{ display: "none" }}>
+        <Page pageNumber={1} />
+      </Document>
+    );
+  } else {
+    displayImg = (
+			withTail ?(
```
Comment (olgafedorenko) [inline src/components/Box/TextBubble.js:148]: make component readable without extra spaces

File: src/components/Box/TextBubble.js [hunk 4]
```diff
@@ -0,0 +1,306 @@
...
+            )}
+            {show ? null : (
+              <span
+                id="bankName"
+                style={{ display: "block", textAlign: "center" }}
+              >
+                {bankName}
+              </span>
+            )}
+            {/* {show ? <Image onError={addDefaultSrc} src='https://cdn.synapsepay.com/bank_logos/new/aaaaa.png' alt="logo" />  : <span id="bankName">{bankName}</span>} */}
```
Comment (olgafedorenko) [inline src/components/Box/TextBubble.js:295]: delete comments

File: src/components/Input/InputSearch.js [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+      clickCount: 0,
+      showCloseIcon: false,
+    };
+  }
+
+  updateValue = (e) => {
+    e.preventDefault();
+    const { clickCount } = this.state;
+    const { onChange } = this.props;
+    console.log('asdf');
```
Comment (olgafedorenko) [inline src/components/Input/InputSearch.js:78]: do you need this console?

File: src/components/Input/InputSearch.js [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+      value,
+      handleSubmit,
+      onChange,
+      onFocus,
+      clientColor,
+			placeholder,
+      fullWidth
+    } = this.props;
+
+    // setTimeout(() => {
```
Comment (olgafedorenko) [inline src/components/Input/InputSearch.js:116]: please delete comments and make the component lines with right spaces

File: src/components/Input/util/voiceIcon.js [hunk 7]
```diff
@@ -0,0 +1,276 @@
...
+//     justify-content: center;
+//     align-items: center;
+//     :hover {
+//       background: ${utils.hoverColor(`#${clientColor}`)};
+//     };
+//   `;
+
+//   return (
+//     <Wrapper>
+//       <svg width="13" height="16" viewBox="0 0 13 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (olgafedorenko) [inline src/components/Input/util/voiceIcon.js:48]: do you need comments in this file?

File: src/components/modalRelated/ModalContent/LoadingModal.js [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+      props.updateModal('isModal', false);
+    }, 200);
+  }
+
+  displayFullButton = () => {
+    const { modalType, clientColor } = this.props;
+    return (
+      <div className="modal-button-container">
+        <Button fullWidth idx="exit-modal-two" handleButtonSubmit={this.closeModal} btnTxt="Cancel" buttonColor={clientColor} />
+        {/* <Button fullWidth id="exit-modal-two" onClick={this.closeModal} buttonColor={clientColor}> Cancel </Button> */}
```
Comment (olgafedorenko) [inline src/components/modalRelated/ModalContent/LoadingModal.js:53]: delete comments if you don't need them

File: src/components/modalRelated/ModalContent/LoadingModal.js [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+    const minutes = parseInt(timer / 60, 10);
+    return (
+      <div>
+        <div
+          style={{ height: '100%', width: '100%', backgroundColor: 'white', paddingTop: '15%', borderRadius: '20px 20px 0 0' }}
+        >
+          <div className="loading_div">
+            <img className="loading_bank_image" alt="bank_logo" />
+            <div style={{ height: '50px', position: 'relative', marginTop: '10%' }}>
+              Please link accounts within
```
Comment (olgafedorenko) [inline src/components/modalRelated/ModalContent/LoadingModal.js:81]: Please update the modal with the latest version what you have and remove  unnecessary comments

File: src/components/modalRelated/ModalContent/EditModal.js [hunk 10]
```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react';
+
+// import { Button } from 'synapsefi-chatbot-ui';
```
Comment (olgafedorenko) [inline src/components/modalRelated/ModalContent/EditModal.js:3]: remove comments

File: src/components/util/VoiceMemo.js [hunk 11]
```diff
@@ -0,0 +1,348 @@
+// // -----------------------------------------------------------------------------------------
+// // ------------------------------------ Library Imports ------------------------------------
+// // -----------------------------------------------------------------------------------------
+
```
Comment (olgafedorenko) [inline src/components/util/VoiceMemo.js:4]: what is the plan with this file?

Comment (vagarii) [reply src/components/util/VoiceMemo.js:4]: last time you said you will continue work on this

File: src/components/Input/InputSearch.js (hunk 6 above)
Comment (vagarii) [reply src/components/Input/InputSearch.js:116]: think all the comments you mentioned about space is github formatting issue, I don't have space issue on my end

**Pull request merged**